Makefile

```diff
@@ -1,12 +1,8 @@
 ORG     ?= $(shell basename $(realpath ..))
 PKGS    := $(shell go list ./... | grep -v /vendor/)
 
-dep: ## Get build dependencies
-	go get -v -u github.com/golang/dep/cmd/dep
-.PHONY: dep		
-
-build: ## Build the app after getting all dependencies (make sure to call make dep before)
-	dep ensure && go build ${TARGETS}
+build:
+	go build ${TARGETS}
```

Comment:
Note that after vgo/go modules, `go build` automatically fetch all the dependencies, as long as you're outside of $GOPATH. No need to run a command like `dep ensure`.

So If you've been checking out helmfile's tree into e.g. `~/go/src/github.com/roboll/helmfile`, move the helmfile dir out of `~/go`, like `~/helmfile`.

Reply:
cc/ @sgandon FYI

Reply:
cool!, that was just all so wierd to have to clone the repo into a specific folder.